packages/legend-graph/src/graphManager/PureGraphManagerPlugin.ts

```diff
@@ -73,4 +77,8 @@ export abstract class PureGraphManagerPlugin extends AbstractPlugin {
    * Get the list of Pure grammar element labelers.
    */
   getExtraPureGrammarElementLabelers?(): PureGrammarElementLabeler[];
+
+  getExtraIdFromTestableGetters?(): IdFromTestableGetter[];
```

Comment:
--> `TestableIDBuilder`

Reply:
Also, let's add doc for these 2

Reply:
These should also go to a separate `DSLData_PureGraphManagerPlugin_Extension`

Reply:
no they should not go to DSLData. They are not part of DSLDATA

Reply:
Test and Data are two different concepts and Testable should not belong under the DSL_Data

Reply:
yep, my bad